libglusterfs/src/glusterfs/glusterfs.h

```diff
@@ -543,6 +543,7 @@ struct _cmd_args {
     int client_pid_set;
     unsigned uid_map_root;
     int32_t lru_limit;
+    int32_t inode_table_size;
```

Comment:
unsigned int, perhaps? Why would we allow negative numbers?

Reply:
Also consistent with the definition in the fuse part:
```
uint32_t inode_table_size
```

Reply:
Most of this was based on lru-limit.  fuse-bridge.c starts with the option being of type GF_OPTION_TYPE_INT, and then over time it's reworked into a uint32_t ... if you prefer unsigned int, I have no objection to making everything unsigned int here, I would suggest that a separate commit is made to also update lru-limit though.

There are a few types that needs to play together from what I understand, since I'm not 100% familiar with all the do's and don'ts I stuck as closely as possible with what lru-limit does as possible.

I think that really both types should be unsigned int though, so can update, but I think should update lru_limit at the same time (or as a previous commit).

Reply:
@mykaul should I proceed to update both?  Does this affect the GF_OPTION_TYPE_ value to be used?

Reply:
Separate commit is fine by me, it was just a nit.